packages/transmute-framework/src/stream-model/__tests__/sync.spec.js

```diff
@@ -100,4 +100,28 @@ describe('sync', () => {
     expect(streamModel2.state.lastIndex).toBe(1);
     // console.log(JSON.stringify(streamModel2.state, null, 2));
   });
+
+  it('calling sync repeatedly returns the same state', async () => {
```

Comment:
To be consistent with other `it`s, change to: "returns the same state if called repeatedly".

Reply:
agree